lib/puppet/provider/rbac_role/ruby.rb

```diff
@@ -64,6 +64,22 @@ def destroy
     fail "The id parameter is read-only."
   end
 
+  def normalize_users(list)
+    users = nil
+    list.collect! do |item|
+      next item if item.to_i != 0
```

Comment:
Is this intended to make it so you can invoke this on a mixed list of IDs and names?

Reply:
Good catch, the users have a UUID and not a number, need to fix...